DependencyInjection/MonologExtension.php

```diff
@@ -468,6 +468,8 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['bubble'],
                 $handler['use_ssl'],
                 $handler['message_format'],
+                $handler['host'],
```

Comment:
@apfelbox you could still not pass default here. You would have to pass a special value supported by Symfony to dump it as the keyword ``default``